db/migrate/20150429072423_add_cache_pattern_to_projects.rb

```diff
@@ -0,0 +1,5 @@
+class AddCachePatternToProjects < ActiveRecord::Migration
+  def change
+    add_column :projects, :cache_pattern, :string, :default => ""
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.